lib/mayaUsd/undo/OpUndoItemList.h

```diff
@@ -52,6 +52,10 @@ class OpUndoItem
     MAYAUSD_CORE_PUBLIC
     virtual ~OpUndoItem() = default;
 
+    /// \brief execute a single sub-operation. By default calls redo.
+    MAYAUSD_CORE_PUBLIC
+    virtual bool execute() { return redo(); }
+
```

Comment:
As explained, the old design assumed redo == execute, but this not the case for some UFE commands.